package.json

```diff
@@ -28,16 +28,18 @@
   "scripts": {
     "audit:all": "npm audit --if-present",
     "audit:fix:all": "npm audit --fix --if-present",
+    "build": "docker compose build",
```

Comment:
I'm not sure on the naming of this as in all other packages we have
`    "build": "rimraf ./public/js && rimraf ./public/css && webpack --mode production --config webpack.prod.config.js",`

which is rather different than building an image?